lib/files.c

```diff
@@ -719,6 +719,9 @@ tool_rc files_save_ESYS_TR(ESYS_CONTEXT *ectx, ESYS_TR handle, const char *path)
 SAVE_TYPE(TPM2B_PUBLIC, public)
 LOAD_TYPE(TPM2B_PUBLIC, public)
 
+SAVE_TYPE(TPMT_PUBLIC, public_area)
```

Comment:
@asac tpmt_public vs public_area, what are your thoughts for the resulting function naming files_save_public_area vs files_save_tpmt_public? 

I was thinking files_save_public and files_save_public_area seem to imply the same thing.

Reply:
Call it template